lib/ansible/cli/galaxy.py

```diff
@@ -402,8 +402,13 @@ def execute_install(self):
             # query the galaxy API for the role data
 
             if role.install_info is not None and not force:
-                display.display('- %s is already installed, skipping.' % role.name)
-                continue
+                if role.install_info['version'] != role.version:
```

Comment:
actually, the change to using force will stop unexpected downgrades. 

You have to pass `--force` to ansible-galaxy to get an upgrade or a downgrade. (This might not be expected but that's the problem with enforcing the use of force just to get an upgrade)